client/src/leap/soledad/client/sync.py

```diff
@@ -39,6 +39,7 @@ class SoledadSynchronizer(Synchronizer):
 
     Also modified to allow for interrupting the synchronization process.
     """
+    received_docs = []
```

Comment:
Does this need to be a class attribute? If not, maybe the place to initialize it should be during object initialization.